### PR TITLE
Fixed #959: add analytics for iopay wallet 

### LINKED ISSUE #959: add analytics for iopay wallet
use UA-111756489-15

### PR DESCRIPTION
**Pull request by nmhung1210**
**What type of PR is this?**
feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #959 

**Special notes for your reviewer**:
This PR has implementation for `pageview` only.

**Does this PR introduce a user-facing change?**:
NONE

### CODE REVIEW

File: src/electron/global-state.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ module.exports = {
       multiChain,
       defaultERC20Tokens,
       analytics: {
-        googleTid: "UA-111756489-2"
+        googleTid: "UA-111756489-15"
```
Comment (puncsky) [inline src/electron/global-state.js]: preserve the current tid for web

Comment (nmhung1210) [reply src/electron/global-state.js]: This file is used for the desktop app only. The web version is using the config from `default.js`.

**Comment by nmhung1210** [comment]
> please resolve comments

resolved

File: src/electron/package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "ioPay",
```
Comment (puncsky) [inline src/electron/package.json]: can we preserve the name "ioPay"?

**Pull request merged**